### ISSUE TITLE
Feature request: Oled display timeout 

### ISSUE THREAD

**Issue by jascdk** [SuperHouse/AirQualitySensorD1Mini#9]
Hi Jon :)

Would it be possible to make some kind of timeout on the oled so the display goes out / maybe dimm after a while. Would prolong the life of the display so it does not burn in . All the best Jacob from Denmark

**Comment by jonoxer** [comment]
Yes, definitely. I thought about that while I was working on it, but I couldn't decide what logic to use for when it should be un-dimmed. Perhaps it should wake up the display when any value changes, or when the button is pressed.

I'd like to add this. Do you have a suggestion for exactly how it should behave?

**Comment by JamesKennewell** [comment]
It would depend on the application its being used for.

in some cases i can see having a button which toggles the oled on and off being helpful. 

or maybe even having the ability to use MQTT to set times the screen is on and off?

Personally if your using the AQS without mqtt then pushing the button to wake the screen up would be preferered and having a timeout of 30 seconds would be enough id think.

**Comment by jascdk** [comment]
Thanks for your quick answers:) Well in my case I think it would be quite useful - as @JamesKennewell - also says - that the button would toggle the screen on and then you should still be able to go through the different pages. If you then stop going through the pages a kind of timer - 30 seconds or so - should then put the oled to sleep again. Just my thought - all solutions are welcome :)
